pmml-evaluator/1.1.16-wso2v1/pom.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Make all the versions to have 1.1.16.wso2v1 inserted of 1.1.16-wso2v1 and also fix other orbits to have proper versions and fix their group id